cmd/helm/install.go

```diff
@@ -304,6 +308,22 @@ func (i *installCmd) run() error {
 		helm.InstallWait(i.wait),
 		helm.InstallDescription(i.description))
 	if err != nil {
+		if i.safe {
+			fmt.Fprintf(os.Stdout, "INSTALL FAILED\nPURGING CHART\nError: %v", prettyError(err))
```

Comment:
One little nit here I just found in testing. Can you add a newline to the end of this error? If something else prints right after, it is mashed up against this error

Reply:
Added this